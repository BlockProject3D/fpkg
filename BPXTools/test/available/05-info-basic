export NAME="Info (Basic)"
export COMMAND="-f test.bpx info"
export DESC="Test the info command"
export STATUS=0