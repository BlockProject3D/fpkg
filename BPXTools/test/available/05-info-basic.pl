$Test = {
    Name => "Info (Basic)",
    Command => "-f test/available/test.bpx info",
    Description => "Test the info command",
    Status => 0
};

sub TestBegin {

}

sub TestEnd {
    return 1;
}
