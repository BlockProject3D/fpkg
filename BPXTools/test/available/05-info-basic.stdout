====> BPX Main Header <====
Type: P
Version: 1
File size: 1629
Number of sections: 2
====> End <====

